python/tk_houdini/utils/parm_template_wrappers.py

```diff
@@ -93,25 +163,60 @@ def index_of_template(self, template_name):
         """
         return self.__child_names.index(template_name)
 
+    def extend(self, parms):
+        """
+        Add multiple templates to the child templates list..
+
+        :param list(Parm)) parms: The list of Parm
+            instances to add.
+        """
+        for parm in parms:
+            self.append(parm)
+
```

Comment:
If you fancy, you can also implement [\_\_iadd\_\_](https://docs.python.org/3/reference/datamodel.html#object.__iadd__) to be the same as `extend()`, allowing you to have the option to `folder += [param1, param2, param3]`

```suggestion

    __iadd__ = extend
```

Reply:
I might leave it for now. It's not really used anywhere

Reply:
ah f*** it lets just put it in